docs/src/content/docs/utilities/Signals/computed-from.md

```diff
@@ -87,28 +90,28 @@ setTimeout(() => {
 The console log will be:
 
 ```ts
--[1, 2] - // initial value
+42 - // initial value
 	3 - // combined value after 1 second
 	5; // combined value after 3 seconds
 ```
 
-As we can see, the first value will not be affected by the rxjs operators, because they are asynchronous and the first value is emitted synchronously.
-In order to change the first value, we can use startWith operator.
+As we can see, we passed an `initialValue` as third argument, to prevent _throwing error_ in case of observable that will **not has sync value** becouse they emit later their values.
```

Comment:
```suggestion
As we can see, we passed an `initialValue` as the third argument, to prevent returning `null` in case of an observable that will **not have a sync value** because they emit later their values.
```